samples/web/blazor-server-application/Core_8/WebApp/Properties/launchSettings.json

```diff
@@ -0,0 +1,35 @@
+{
+  "iisSettings": {
+    "iisExpress": {
+      "applicationUrl": "http://localhost:22439",
+      "sslPort": 44348
+    }
+  },
```

Comment:
```suggestion
```

Reply:
I don't quite understand why this would need to be done.  Not only that, but I now have warnings that `the object is unclosed, '}' expected`.  This was also a file that was generated from the default Blazor project.

Reply:
Yeah, I see. It's a relic from my past, you can ignore me 😬